qttypes/src/lib.rs

```diff
@@ -666,6 +749,265 @@ impl QVariant {
         })
     }
 
+    /// Wrapper around [`toByteArray()`][method] method.
+    ///
+    /// [method]: https://doc.qt.io/qt-5/qvariant.html#toByteArray
+    pub fn to_qbytearray(&self) -> QByteArray {
+        cpp!(unsafe [self as "const QVariant*"] -> QByteArray as "QByteArray" {
+            return self->toByteArray();
+        })
+    }
+
+    /// Returns the variant as a Rust's native [`Vec`]`<`[`u8`]`>` if the variant is supported by
+    /// [`to_qbytearray()`] method, otherwise returns an empty vector.
+    ///
+    /// Unlike [`to_qbytearray()`] which might use [implicit sharing] (copy-on-write), this method
+    /// always allocates memory for a [`Vec`].
+    ///
+    /// [`to_qbytearray()`]: Self::to_qbytearray()
+    /// [implicit sharing]: https://doc.qt.io/qt-5/implicit-sharing.html
+    pub fn to_bytes(&self) -> Vec<u8> {
```

Comment:
Is there a particular use case for this function?

Reply:
Idk, just a shortcut, a convenience. Kinda making it more Rust-oriented and usable for non-Qt code.

I wanted to name it `to_vec`, but it would cause all sorts of conflicts with `toList() -> QVariantList`.